apps/bluetooth/README.md

```diff
@@ -0,0 +1,7 @@
+## Bluetooth
+
+Bluetooth app is responsible for discover and pair Bluetooth devices.
```

Comment:
It's better to say:
Bluetooth app is responsible for:
1. Pair Bluetooth devices.
2. Discover Bluetooth devices for user picked. Then, send the file(s) to the paired device. Otherwise, send the file(s) after inline-pairing with the unpaired device successfully.